app/assets/stylesheets/active_admin/_forms.scss

```diff
@@ -249,80 +213,81 @@ form {
 $sidebar-inner-content-width: $sidebar-width - ($section-padding * 2);
 
 .sidebar_section {
-
-  label {
-    display: block;
-    text-transform: uppercase;
-    color:  $form-label-color;
-    font-size: 0.9em;
-    font-weight: bold;
-  }
-
   select {
     width: $sidebar-inner-content-width;
   }
 
-  input[type=text], input[type=password], input[type=email], input[type=url], input[type=tel], textarea {
+  input[type=text],
+  input[type=password],
+  input[type=email],
+  input[type=url],
+  input[type=tel],
+  textarea {
     width: $sidebar-inner-content-width - ($text-input-horizontal-padding * 2);
   }
-
 }
 
 // -------------------------------------- Filter Forms
 
 $filter-field-seperator-width: 12px;
 
-$side-by-side-filter-input-width: ($sidebar-inner-content-width / 2) - ($text-input-horizontal-padding * 2) - $filter-field-seperator-width;
-$side-by-side-filter-select-width: ($sidebar-inner-content-width / 2) - $filter-field-seperator-width;
+$side-by-side-filter-input-width: ($sidebar-inner-content-width / 2) - $filter-field-seperator-width;
+$side-by-side-filter-select-width: ($sidebar-inner-content-width / 2);
 
-$date-range-filter-input-right-padding: 27px;
-$date-range-filter-input-horizontal-padding: $date-range-filter-input-right-padding + $text-input-horizontal-padding;
-$date-range-filter-input-width: ($sidebar-inner-content-width / 2) - $filter-field-seperator-width - $date-range-filter-input-horizontal-padding;
+$date-range-filter-input-right-padding: 25px;
 
-form.filter_form {
-  .filter_form_field {
-    margin-bottom: 10px;
-    clear: both;
+$date-range-filter-input-width: ($sidebar-inner-content-width / 2) - ($filter-field-seperator-width / 2);
 
-    &.select_and_search {
-      input[type=text] {
-        margin-left: $filter-field-seperator-width + 4;
-        width: $side-by-side-filter-input-width;
-      }
-      select {
-        width: $side-by-side-filter-select-width;
-      }
+.filter_form_field {
+  margin-bottom: 12px;
+
+  .label {
+    display: block;
+    margin: 0 0 0.18em 0;
+  }
+
+  &.select_and_search {
+    input[type=text] {
+      margin-left: $filter-field-seperator-width;
+      width: $side-by-side-filter-input-width;
     }
 
-    &.filter_check_boxes {
-      label { margin-bottom: 3px; }
-      fieldset {
-        margin-bottom: 0px;
-        padding-bottom: 0px;
-      }
-      .check_boxes_wrapper label {
-        font-weight: normal;
-        margin-bottom: 3px;
-        text-transform: none;
-        font-size: 1.0em;
-        input { vertical-align: baseline; }
-      }
+    select {
+      width: $side-by-side-filter-select-width;
     }
+  }
 
-    &.filter_date_range {
-      .seperator {
-        display: inline-block;
-        text-align: center;
-        width: $filter-field-seperator-width;
-      }
+  &.filter_date_range {
+    input {
+      background: #fff image-url('active_admin/datepicker/datepicker-input-icon.png') no-repeat 100% 6px;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Prefer double-quoted strings